Dockerfile.reva

```diff
@@ -20,5 +20,5 @@ FROM golang:1.13
 
 WORKDIR /go/src/github/cs3org/reva
 COPY . .
-RUN make build-reva-docker && cp /go/src/github/cs3org/reva/cmd/reva/reva /go/bin/reva
+RUN make build-reva-docker && cp /go/src/github/cs3org/reva/cmd/revad/reva /go/bin/reva
```

Comment:
What is the problem you are trying to solve? This Dockerfile is for the client cli tool (reva) and not for the server (revad)

Reply:
Fixed error build docker image from command line, 
issue: https://github.com/cs3org/reva/issues/530

After fix, build om localmachine works at commands:

```
docker build -t reva -f Dockerfile.reva .
``` 
 or
 
```
git clone https://github.com/JarCz/reva.git
cd reva
cp Dockerfile.reva Dockerfile
docker build -t reva .
```